### DISCUSSION TITLE
Plugin version 1.8.5 incompatible with netbox 3.5.8

### DISCUSSION THREAD

**Discussion by pmarchal25** [Onemind-Services-LLC/netbox-secrets#94] [General]
Hi everyone, i have just updated my netbox from the version 3.5.7 to the version 3.5.8 and i am trying to install the netbox secrets plugin (i know how to do it because i tried to do it in the latest version) but when I try to install the last version of the pluggin (1.8.5) this error appears:

**### ERROR: Could not find a version that satisfies the requirement netbox-secretstore==1.8.5 (from versions: 1.0, 1.0.1, 1.0.3, 1.0.4, 1.0.5, 1.0.6, 1.0.8, 1.0.9, 1.0.10, 1.0.11, 1.0.14, 1.0.16, 1.2.0, 1.3.0, 1.4.0, 1.4.1)
ERROR: No matching distribution found for netbox-secretstore==1.8.5**

Maybe this plugin version is incompatible with the 3.5.8 netbox version?

Best regards and Thank you for your time.

Pedro

**Comment by abhi1693** [comment]
Our plugin is `netbox-secrets`
